### ISSUE TITLE
ellipsis truncate html code for quote in firefox

### ISSUE THREAD

**Issue by mikesmichael** [lentschi/ngx-ellipsis#29]
Firefox version: 70.0.1
ngx-ellipsis version: 2.0.9

My user run into a weird issue. ellipsis is truncating in the middle of the html code for a quote. I'm not able to reproduce with chrome.

Source to reproduce:
```
<div [ngStyle]="{'max-height.em': 1.2 }" ellipsis ellipsis-content="C'est l'homme qui a vu l'homme qui a vu l'ours."></div>
```
Output:

![image](https://user-images.githubusercontent.com/9534827/68878550-ec458080-06d5-11ea-86b6-b2aae5f7ba33.png)

**Comment by lentschi** [comment]
This is due to the fix in #10 - I need to check that the html special characters are never torn apart.

**Comment by lentschi** [comment]
@mikesmichael Thanks a lot for reporting this issue and sorry for not reacting for such a long time! (I had overlooked the issue completely.)
This is now fixed in `ngx-ellipsis` version 2.1.0.

**Comment by mikesmichael** [comment]
Thanks a lot!! I haven't expect a quick reaction. Less then 2 weeks for a fix is enough quick for me!

Great feature by the way!
